client/scripts/views/components/social_sharing_carat.ts

```diff
@@ -0,0 +1,41 @@
+import m from 'mithril';
+import { MenuItem, Icons, PopoverMenu, Icon } from 'construct-ui';
+import { int } from 'aws-sdk/clients/datapipeline';
+
+export const SocialSharingCarat: m.Component<{
+    commentID?: int,
+  }> = {
+    view: (vnode) => {
+      return m(PopoverMenu, {
+        transitionDuration: 0,
+        closeOnOutsideClick: true,
+        closeOnContentClick: true,
+        menuAttrs: { size: 'default' },
+        content: [
+          m(MenuItem, {
+            iconLeft: Icons.COPY,
+            label: 'Copy URL',
+            onclick: async (e) => {
+              if (vnode.attrs.commentID == null) {
```

Comment:
this check can be done either `if (vnode.attrs.commentID)` or the "not" version `if (!vnode.attrs.commentID)`. If it's undefined (nothing passed in), those conditions will be false/true respectively.